### PR TITLE
Dev 12639 es indices

### PR DESCRIPTION
**Pull request by IsraelMurciano**

### CODE REVIEW

File: config/crd/bases/mlops.cnvrg.io_cnvrgapps.yaml [hunk 1]
```diff
@@ -90,6 +90,8 @@ spec:
                       type: boolean
                     cnvrgJobUid:
                       type: string
+                    cnvrgPrivilegedJob:
```
Comment (Dimss) [inline config/crd/bases/mlops.cnvrg.io_cnvrgapps.yaml:93]: merge into your branch latest main

File: chart/crds/mlops.cnvrg.io_cnvrgapps.yaml [hunk 2]
```diff
@@ -90,6 +90,8 @@ spec:
                       type: boolean
                     cnvrgJobUid:
                       type: string
+                    cnvrgPrivilegedJob:
```
Comment (Dimss) [inline chart/crds/mlops.cnvrg.io_cnvrgapps.yaml:93]: this has been removed from main, merge main into your branch

File: pkg/controlplane/tmpl/crds/mlops.cnvrg.io_cnvrgapps.yaml [hunk 3]
```diff
@@ -90,6 +90,8 @@ spec:
                       type: boolean
                     cnvrgJobUid:
                       type: string
+                    cnvrgPrivilegedJob:
```
Comment (Dimss) [inline pkg/controlplane/tmpl/crds/mlops.cnvrg.io_cnvrgapps.yaml:93]: merge into your branch latest main

**Review by Dimss — CHANGES_REQUESTED** [review]

**Review by Dimss — APPROVED** [review]

File: pkg/controlplane/tmpl/webapp/dep.tpl [hunk 4]
```diff
@@ -101,6 +101,8 @@ spec:
         env:
         - name: "CNVRG_RUN_MODE"
           value: "webapp"
+        - name: "CNVRG_ENDPOINTS_INDEX"
```
Comment (itayariel) [inline pkg/controlplane/tmpl/webapp/dep.tpl:104]: @IsraelMurciano this env is only getting into the app pod and not to any of the sidekiq pods
reopening issue, please fix, maybe use one of the configmaps that are mounted to those pods

**Pull request merged**